lib/analytics.js

```diff
@@ -633,6 +627,32 @@ Analytics.prototype.noConflict = function(){
 };
 
 /**
+ * Accepts a page call's `.properties` object and transforms it into a
+ * spec-compliant `context.page` object.
+ *
+ * @param {Object} properties
+ * @return {Object}
+ */
+
+function pageContext(properties){
+  properties = properties || {};
+
+  // https://segment.com/docs/spec/page/#properties
+  var reservedProperties = {
+    path: canonicalPath(),
+    referrer: document.referrer,
+    search: location.search,
+    title: document.title,
+    url: canonicalUrl(properties.search || location.search)
```

Comment:
Remove https://github.com/segmentio/analytics.js/blob/feat/context.page/test/analytics.js#L420 test case when nixing this line